packages/default/scss/scrollview/_layout.scss

```diff
@@ -4,6 +4,8 @@
 @include exports('scrollview/layout') {
     $scrollview-page-button-size: 8px;
     $scrollview-arrow-icon-size: 4.5em;
+    $scrollview-pager-spacing-x: $padding-y;
+    $scrollview-pager-spacing-y: 10px;
```

Comment:
Don't you find it odd to have unused variables in the file? :smile: The variables are the public API of the theme. Changing them should allow users of the theme to change its properties. Having a variable that is unused is as frustrating as a component property that does nothing.

With my previous comment, I meant that introducing such variables in theme-default will allow you to remove the layout styles from theme-material.

To illustrate, instead of this...

```scss
// default/component/_layout.scss
.foo { padding: 10px; }

// material/component/_layout.scss
@import "default/component/_layout.scss";
.foo { padding: 16px; }
```
... you can use variables:

```scss
// default/component/_layout.scss
$foo-padding: 10px !default;
.foo { padding: $foo-padding; }

// material/component/_layout.scss
$foo-padding: 16px !default;
@import "default/component/_layout.scss";
```

The latter is preferred, because it simply changes the default value for theme-material, without the need to write extra CSS. This allows us to have less code to maintain, and also reduces the CSS output for the derived theme.